tasks/prime.py

```diff
@@ -409,17 +409,19 @@ def create_move_task_order(self):
                     "lastName": "Swinglehurst-Walters",
                     "agency": "MARINES",
                     "email": "swinglehurst@example.com",
+                    "rank": "E_3",
```

Comment:
It looks like we need to set a `"dodID"` in the Customer as well - I added it in the branch to test.

Reply:
Okay thanks!